### PR TITLE
DOCSP-13662: v0.7.0 updates

### PR DESCRIPTION
**Pull request by steveren**
Jira:
https://jira.mongodb.org/browse/DOCSP-13662
https://jira.mongodb.org/browse/DOCSP-13741
https://jira.mongodb.org/browse/DOCSP-13749
https://jira.mongodb.org/browse/DOCSP-13739

Staged:
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-13662/

### CODE REVIEW

File: source/reference/methods.txt [hunk 1]
```diff
@@ -412,12 +412,17 @@ Cursor Methods
...
        .. arrayAccess
 
-   * - :method:`cursor.batchSize()`
+   * - ``cursor.batchSize()``
 
      - Controls the number of documents MongoDB will return to the
        client in a single network message.
 
-       .. clone
+       In the following example, the query returns results in batches of
```
Comment (jeff-allen-mongo) [inline source/reference/methods.txt:420]: [nit] Consider tightening this up to "The following example query returns..."

File: source/reference/methods.txt [hunk 2]
```diff
@@ -709,6 +714,24 @@ Native Methods
...
+       .. code-block:: code
+
+          > print("hello world")
+          hello world
+
+          > x = "example text"
+          > print(x)
+          example text
+
+   * - :method:`sleep()`
```
Comment (jeff-allen-mongo) [inline source/reference/methods.txt:731]: This links me to the top of the current page. I assume this isn't intentional?

File: source/run-commands.txt [hunk 3]
```diff
@@ -47,6 +47,22 @@ the server.
    provides consistent behavior by always throwing an exception in these
    scenarios.
 
+Shell Commands
```
Comment (jeff-allen-mongo) [inline source/run-commands.txt:50]: I think "shell commands" is easy to conflate with "shell methods". What do you think about making the heading the specific task this command accomplishes, like "Clear ``mongosh`` Console" (or something like that), and explaining the ``cls`` method that way.

If we end up with more commands that would fit in here, we can consider a table structure at a later time.

File: source/install.txt [hunk 4]
```diff
@@ -101,6 +101,11 @@ Procedure
       
             .. include:: /includes/steps/install-shell-linux-generic.rst
 
+.. important:: Windows Users
```
Comment (jeff-allen-mongo) [inline source/install.txt:104]: I think we can move this so it only appears under the Windows tab.

File: source/install.txt [hunk 5]
```diff
@@ -101,6 +101,11 @@ Procedure
...
+
+   ``mongosh`` preferences are stored by default in the directory
```
Comment (jeff-allen-mongo) [inline source/install.txt:106]: As part of the same ticket (DOCSP-13741), there is also an update to where log files are stored. We may also need to update https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-13662/logs accordingly?

**Review by jeff-allen-mongo — COMMENTED** [review]
Thanks Steve! Left a few comments / suggestions.

File: source/install.txt (hunk 4 above)
Comment (jeff-allen-mongo) [reply source/install.txt:104]: Do we need a parallel blurb for Linux / macos tabs which explain where to find Preferences on those operating systems as well?

Comment (steveren) [reply source/install.txt:109]: Moved it under the windows tab and changed it to a note. Judging from the Jira ticket, this is a behavioral oddity which is only noteworthy for windows users.

File: source/reference/methods.txt (hunk 1 above)
Comment (steveren) [reply source/reference/methods.txt]: Done.

File: source/reference/methods.txt (hunk 2 above)
Comment (steveren) [reply source/reference/methods.txt]: Fixed.

File: source/run-commands.txt (hunk 3 above)
Comment (steveren) [reply source/run-commands.txt]: Done.

File: source/install.txt [hunk 6]
```diff
@@ -36,6 +36,11 @@ Procedure
 
       .. include:: /includes/steps/install-shell-windows.rst
 
+      .. note::
```
Comment (jeff-allen-mongo) [inline source/install.txt:39]: Would it not be better to have this blurb appear under the macOS and Linux tabs, and have the Windows-specific blurb under the Windows tab?

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM with one suggested fix.

File: source/install.txt (hunk 6 above)
Comment (steveren) [reply source/install.txt:39]: Fixed.

**Review by aherlihy — APPROVED** [review]

**Review by addaleax — APPROVED** [review]

**Pull request merged**